src/main/java/gov/cabinetoffice/gap/adminbackend/services/ApplicationFormService.java

```diff
@@ -327,7 +334,7 @@ public void patchApplicationForm(Integer applicationId, ApplicationFormPatchDTO
                 application.setLastPublished(Instant.now());
             }
 
-            ApplicationFormUtils.updateAuditDetailsAfterFormChange(application, session);
+            ApplicationFormUtils.updateAuditDetailsAfterFormChange(application, session, isLambdaCall);
```

Comment:
What happens if session is null?

Reply:
`session` will only be `null` when isLambdaCall is true. In that case we skip `setLastUpdateBy` (because the lambda is updating the application) There's a test which asserts that `setLastUpdateBy` isn't called when `isLambdaCall` is true